docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -354,6 +354,37 @@ pipelines:
 | **results in pull request comments** | ❌ not available |
 | **automatic CI setup** | ❌ not available |
 
+## Azure
+
+```yaml
+# trigger:
+#  - master
+
+pool:
+  vmImage: ubuntu-latest
+# variables:
+# - group: Semgrep app token group
+
+steps: 
+
+- script: |
+    python -m pip install --upgrade pip
+    pip install semgrep
+    semgrep ci --config auto
+  env: 
+    SEMGREP_PR_ID: $(System.PullRequest.PullRequestNumber)
+```
+
+### Feature support
+
+| Feature | Status |
+| --- | --- |
+| **diff-aware scanning** | ✅ [configure manually](configuration-reference.md#diff-aware-scanning-semgrep_baseline_ref) |
+| **hyperlinks in Semgrep App** | ✅ [configure manually](configuration-reference.md#get-hyperlinks-in-semgrep-cloud) |
+| **results in native dashboard** | 💢 not applicable |
+| **results in pull request comments** | ❌ not available |
```

Comment:
@holdenmcg I'll approve this, but I am letting you know in advance that I'm planning to remove "results in pull request comments" because it's dependent on SCM, not CI provider. I think we got into the habit of updating a legacy (outdated) table 😅